lib/ui/pages/detail_page.dart

```diff
@@ -0,0 +1,306 @@
+part of 'pages.dart';
+
+class DetailPage extends StatelessWidget {
+  const DetailPage({Key? key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return SafeArea(
+        child: Scaffold(
+            backgroundColor: AppColor.darkBlueColor,
+            body: Stack(
+              children: [
+                Container(
+                  height: 200,
+                  width: double.infinity,
+                  decoration: const BoxDecoration(
+                      image: DecorationImage(
+                          image: NetworkImage(
+                            "https://images.unsplash.com/flagged/photo-1557804521-990f076ccb96?ixlib=rb-4.0.3&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1171&q=80",
+                          ),
+                          fit: BoxFit.cover)),
+                ),
+                CustomScrollView(
+                  anchor: 0.2,
+                  slivers: [
+                    SliverToBoxAdapter(
+                      child: SizedBox(
+                        height: 40,
+                        child: Column(
+                          mainAxisAlignment: MainAxisAlignment.end,
+                          children: <Widget>[
+                            Container(
```

Comment:
Since we have just one widget inside the `Column` children and the `Container` has a `height` parameter, we can remove the `SizedBox` and `Column` widgets. Using `Container` as a child in `SliverToBoxAdapter` would be more effective.

Reply:
Alright, I will change it, thanks for the input.